### PR TITLE
ENH: Allow FOV adjustment without slice view reorientation

### PR DESCRIPTION
**Pull request by riep**
The useful FitSliceToAll function in ApplicationLogic may change the slice view orientation. User may want to keep the current orientation and simply adjust the FOV.

### CODE REVIEW

File: Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx [hunk 1]
```diff
@@ -514,7 +514,7 @@ void vtkMRMLApplicationLogic::PropagatePlotChartSelection()
 }
 
 //----------------------------------------------------------------------------
-void vtkMRMLApplicationLogic::FitSliceToAll(bool onlyIfPropagateVolumeSelectionAllowed /* =false */)
+void vtkMRMLApplicationLogic::FitSliceToAll(bool onlyIfPropagateVolumeSelectionAllowed /* =false */, bool rotationAllowed /* =true */)
```
Comment (cpinter) [inline Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx:517]: `rotationAllowed` is not a good name here. It is not about it being allowed (as opposed to the other argument, which handles the case when slice composite nodes do not allow propagate volume selection), but being applied. I suggest `resetOrientation` instead.

**Comment by cpinter** [comment]
Thanks! I added one comment, otherwise it looks good to me.

File: Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx (hunk 1 above)
Comment (riep) [reply Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx]: Done

**Review by pieper — APPROVED** [review]
Looks good to me 👍

**Pull request merged**